modules/softwaremanagementplans/reading.md

```diff
@@ -27,4 +28,4 @@ Here is an (incomplete) list:
   - Deployment
   - Developer
 - [Citation](https://the-turing-way.netlify.app/communication/citable/citable-cff.html#cm-citable-cff)
-- [Software licensing](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html) and [compatibility](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-compatibility.html)
\ No newline at end of file
+- [Software licensing](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html) and [compatibility](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-compatibility.html)
```

Comment:
The first link is not working for me.I like that you added these resources, should they also be part of the licensing module? @LourensVeen